docs/source/user-guide/vision/VisionData.rst

```diff
@@ -37,11 +37,12 @@ Common Class Parameters
 
 To see all other class parameters, see the :class:`VisionData <deepchecks.vision.VisionData>` API reference.
 
-.. _vision_data__creating_vision_data:
 Creating a VisionData Object
 ============================
 In the sub-sections below we will go over three different ways to create a VisionData object:
-1. From a Generic Generator 2. From a PyTorch DataLoader 3. From a TensorFlow Dataset.
+1. From a `Generic Generator <#from-a-generic-generator>`__
+2. From a `PyTorch DataLoader <#from-pytorch-dataloader>`__
+3. From a `TensorFlow Dataset <#from-tensorflow-dataset>`__
```

Comment:
```suggestion
1. `From a Generic Generator <#from-a-generic-generator>`__
2. `From a PyTorch DataLoader <#from-pytorch-dataloader>`__
3. `From a TensorFlow Dataset <#from-tensorflow-dataset>`__
```
Think it's clearer that the link is to the section and not to an explanation about that "object type"